modules/olmv1-installing-an-operator.adoc

```diff
@@ -142,48 +160,90 @@ clusterextension.olm.operatorframework.io/pipelines-operator created
+      reason: Success
+      status: "True"
+      type: Installed
+    - lastTransitionTime: "2024-06-10T17:50:58Z"
+      message: ""
+      observedGeneration: 1
+      reason: Deprecated
+      status: "False"
+      type: Deprecated
+    - lastTransitionTime: "2024-06-10T17:50:58Z"
+      message: ""
+      observedGeneration: 1
+      reason: Deprecated
+      status: "False"
+      type: PackageDeprecated
+    - lastTransitionTime: "2024-06-10T17:50:58Z"
+      message: ""
+      observedGeneration: 1
+      reason: Deprecated
+      status: "False"
+      type: ChannelDeprecated
+    - lastTransitionTime: "2024-06-10T17:50:58Z"
+      message: ""
+      observedGeneration: 1
+      reason: Deprecated
+      status: "False"
+      type: BundleDeprecated
+    installedBundle:
+      name: openshift-pipelines-operator-rh.v1.14.4
+      version: 1.14.4
+    resolvedBundle:
+      name: openshift-pipelines-operator-rh.v1.14.4
+      version: 1.14.4
+kind: List
 metadata:
-  annotations:
-    kubectl.kubernetes.io/last-applied-configuration: |
-      {"apiVersion":"olm.operatorframework.io/v1alpha1","kind":"ClusterExtension","metadata":{"annotations":{},"name":"pipelines-operator"},"spec":{"channel":"latest","packageName":"openshift-pipelines-operator-rh","version":"1.11.x"}}
-  creationTimestamp: "2024-01-30T20:06:09Z"
-  generation: 1
-  name: pipelines-operator
-  resourceVersion: "44362"
-  uid: 4272d228-22e1-419e-b9a7-986f982ee588
-spec:
-  channel: latest
-  packageName: openshift-pipelines-operator-rh
-  upgradeConstraintPolicy: Enforce
-  version: 1.11.x
-status:
-  conditions:
-  - lastTransitionTime: "2024-01-30T20:06:15Z"
-    message: resolved to "registry.redhat.io/openshift-pipelines/pipelines-operator-bundle@sha256:e09d37bb1e754db42324fd18c1cb3e7ce77e7b7fcbf4932d0535391579938280"
-    observedGeneration: 1
-    reason: Success
-    status: "True"
-    type: Resolved
-  - lastTransitionTime: "2024-01-30T20:06:31Z"
-    message: installed from "registry.redhat.io/openshift-pipelines/pipelines-operator-bundle@sha256:e09d37bb1e754db42324fd18c1cb3e7ce77e7b7fcbf4932d0535391579938280"
-    observedGeneration: 1
-    reason: Success
-    status: "True"
-    type: Installed
-  installedBundleResource: registry.redhat.io/openshift-pipelines/pipelines-operator-bundle@sha256:e09d37bb1e754db42324fd18c1cb3e7ce77e7b7fcbf4932d0535391579938280
-  resolvedBundleResource: registry.redhat.io/openshift-pipelines/pipelines-operator-bundle@sha256:e09d37bb1e754db42324fd18c1cb3e7ce77e7b7fcbf4932d0535391579938280
+  resourceVersion: ""
 ----
+where:
+
+`spec.channel`:: Displays the channel defined in the CR of the extension.
+`spec.version`:: Displays the version or version range defined in the CR of the extension.
+`status.conditions`:: Displays information about the status and health of the extension.
+`type: Deprecated`:: Displays if the extension is deprecated. The value of `False` in the `status` field indicates that the extension is not deprecated.
```

Comment:
```suggestion
`type: Deprecated`:: Displays if the extension is deprecated. The value of `False` in the `status` field indicates that the bundle which is to be installed is not deprecated.
```

I'm not sure if "extension" is deprecated would be right. It's more on the lines of - while extension tries to install a bundle, the bundle is itself deprecated. I'll leave the wording to you, as you feel it fit :)

Reply:
For `type: Deprecated`, could we somehow say it's a rollup of `PackageDeprecated`, `ChannelDeprecated`, and `BundleDeprecated`, and let the other descriptions do the heavy lifting?

Reply:
@varshaprasad96 would something like this make sense?
```suggestion
`type: Deprecated`:: Displays if the extension is deprecated. The value of `False` in the `status` field indicates that the resolved bundle is not deprecated.
```